salt/modules/boto_efs.py

```diff
@@ -131,6 +131,7 @@ def _get_conn(key=None,
 
 def create_file_system(name,
                        performance_mode='generalPurpose',
+                       creation_token=None,
```

Comment:
Can you move this new kwarg to the end of the list? We don't want to cause an API change and break functions that may be using position options.